ci/cpu/custreamz/build_custreamz.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
You need something that actually calls this script now. See `ci/cpu/build.sh` and `ci/cpu/upload_anaconda.sh`

Reply:
Got it, thanks! Just fixed this.